### PR TITLE
Tile ghost tint customization

### PR DESCRIPTION
**Pull request by setazer**

### CODE REVIEW

**Comment by raiguard** [comment]
Thanks!

File: changelog.txt [hunk 1]
```diff
@@ -1,4 +1,9 @@
 ---------------------------------------------------------------------------------------------------
+Version: 1.0.3
```
Comment (raiguard) [inline changelog.txt:2]: Please move the changelog to the 1.0.2 entry and keep the date as unknown. My publishing workflow will insert the date and bump the version numbers automatically.

File: info.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "TintedGhosts",
-  "version": "1.0.2",
+  "version": "1.0.3",
```
Comment (raiguard) [inline info.json:3]: Please revert this to 1.0.2, as that is the current development version.

File: settings.lua [hunk 3]
```diff
@@ -3,6 +3,12 @@ data:extend({
     type = "color-setting",
     name = "tg-ghost-tint",
     setting_type = "startup",
-    default_value = { g = 1, b = 1, a = 0.3 },
+    default_value = {r = 0.3, g = 0.6, b = 0.6, a = 0.3},
```
Comment (raiguard) [inline settings.lua:6]: I prefer the old default tint, please change it back.

**Review by raiguard — CHANGES_REQUESTED** [review]
Thanks! Just a few things.

**Comment by setazer** [comment]
gonna reopen this PR with proper flow

**Pull request closed without merging**